packages/volto/news/6082.breaking

```diff
@@ -0,0 +1 @@
+Wrap `Table of contents` block in a `nav` tag for better representation improving the block accessibility. @Tishasoumya-02 @ichim-david
```

Comment:
ToC is not an inline literal, and I reckon the change log should include what it changed _from_ instead of just _to_.

```suggestion
Wrap Table of Contents block in a `nav` tag, instead of a `div` tag, for better representation improving the block accessibility. @Tishasoumya-02 @ichim-david
```